### DISCUSSION TITLE
PUBBLICAZIONE PAGELLE GENITORI E ALUNNI

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#108] [Idee e proposte]
Buonasera ,sarebbe possibile far modo che i genitori/alunni visualizzino le pagelle ,certificato delle competenze o il certificato dell'esame di stato?

**Comment by trinko** [comment]
I genitori visualizzano gli esiti finali con la comunicazione dei voti (documento in PDF). Non è gestito attualmente il formato della pagella standard.
La certificazione delle competenze viene generata e poi gestita lato segreteria.
Altre certificazioni non sono attualemente gestite.
